src/functions/createProduct/handler.ts

```diff
@@ -0,0 +1,66 @@
+// import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse, ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+const AWS = require('aws-sdk');
+const dynamo = new AWS.DynamoDB.DocumentClient();
+import { v4 as uuidv4 } from 'uuid';
+import schema from './schema';
+
+const create = async ( data)=>{
+  const id = uuidv4();
+
+  if (!data.title || !data.price || !data.description || !data.count) {
+    return null;
+  }
+
+  await dynamo.transactWrite({
+    TransactItems: [
+      {
+        Put: {
+          TableName: process.env.STOCKS_TABLE,
+          Item: {
+            product_id: id,
+            count: data.count,
+          },
+        },
+      },
+      {
+        Put: {
+          TableName: process.env.PRODUCTS_TABLE,
+          Item: {
+            id: id,
+            description: data.description,
+            price: data.price,
+            title: data.title,
+          },
+        },
+      },
+    ],
+  }).promise();
+
+  return { ...data, id: id};
+
+}
+
+const createProduct: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+
+  const {productData} = event.body
```

Comment:
Could you please advice what am I doing wrong here?
I'm trying to create a new product using Postman, but I'm getting 'Internal server error'

![image](https://user-images.githubusercontent.com/4612163/224007447-709840f9-ed4e-4083-9a28-7063135f00f3.png)

Reply:
oh yes, you should wrap the info in a "productData" object :)